encoding/codecv3/codecv3.go

```diff
@@ -122,6 +132,11 @@ func ConstructBlobPayload(chunks []*encoding.Chunk, useMockTxData bool) (*kzg484
 	return codecv2.ConstructBlobPayload(chunks, useMockTxData)
 }
 
+// DecodeTxsFromBlob decodes txs from blob bytes and writes to chunks
+func DecodeTxsFromBlob(blob *kzg4844.Blob, chunks []*codecv2.DAChunkRawTx) error {
```

Comment:
```suggestion
func DecodeTxsFromBlob(blob *kzg4844.Blob, chunks []*DAChunkRawTx) error {
```

Reply:
same as https://github.com/scroll-tech/da-codec/pull/10#discussion_r1690992283, can remove this function.